src/v2/guide/plugins.md

```diff
@@ -4,78 +4,78 @@ type: guide
 
-2. Add one or more global assets: directives/filters/transitions etc. e.g. [vue-touch](https://github.com/vuejs/vue-touch)
+2. Fügt ein oder mehrere globale Assets hinzu: Direktiven/Filter/Transitionen usw.; z.B. [vue-touch](https://github.com/vuejs/vue-touch)
 
-3. Add some component options by global mixin. e.g. [vue-router](https://github.com/vuejs/vue-router)
+3. Fügt über ein globales Mixin Komponentenoptionen hinzu; z.B. [vue-router](https://github.com/vuejs/vue-router)
 
-4. Add some Vue instance methods by attaching them to Vue.prototype.
+4. Fügt Vue-Instanzmethoden hinzu, indem sie an `Vue.prototype` angehängt werden.
 
-5. A library that provides an API of its own, while at the same time injecting some combination of the above. e.g. [vue-router](https://github.com/vuejs/vue-router)
+5. Eine Bibliothek mit einer eigenen API, die gleichzeitig aber noch eine Kombination der obigen Punkte umsetzt; z.B. [vue-router](https://github.com/vuejs/vue-router)
 
-A Vue.js plugin should expose an `install` method. The method will be called with the `Vue` constructor as the first argument, along with possible options:
+Ein Vue.js-Plugin sollte eine `install`-Methode aufweisen. Diese Methode wird von Vue mit dem `Vue`-Konstruktor als erstem Argument aufgerufen, zusammen mit optionalen `options`:
 
 ``` js
 MyPlugin.install = function (Vue, options) {
-  // 1. add global method or property
+  // 1. Füge globale Methoden oder Eigenschaften hinzu
   Vue.myGlobalMethod = function () {
-    // something logic ...
+    // beliebige Programmlogik ...
   }
 
-  // 2. add a global asset
+  // 2. Füge ein oder mehrere globale Assets hinzu
   Vue.directive('my-directive', {
     bind (el, binding, vnode, oldVnode) {
-      // something logic ...
+      // beliebige Programmlogik ...
     }
     ...
   })
 
-  // 3. inject some component options
+  // 3. Füge Komponentenoptionen hinzu
   Vue.mixin({
     created: function () {
-      // something logic ...
+      // beliebige Programmlogik ...
     }
     ...
   })
 
-  // 4. add an instance method
+  // 4. Füge eine Instanzmethode hinzu
   Vue.prototype.$myMethod = function (methodOptions) {
-    // something logic ...
+    // beliebige Programmlogik ...
   }
 }
 ```
 
-## Using a Plugin
+## Ein Plugin verwenden
 
-Use plugins by calling the `Vue.use()` global method:
+Verwende ein Plugin durch Aufrufen der globalen Methode `Vue.use()`:
 
 ``` js
-// calls `MyPlugin.install(Vue)`
+// ruft `MyPlugin.install(Vue)` auf
 Vue.use(MyPlugin)
 ```
 
-You can optionally pass in some options:
+So kannst Du `options` übergeben:
 
 ``` js
 Vue.use(MyPlugin, { someOption: true })
 ```
 
-`Vue.use` automatically prevents you from using the same plugin more than once, so calling it multiple times on the same plugin will install the plugin only once.
+`Vue.use` hindert Dich automatisch daran, dasselbe Plugin mehr als einmal zu verwenden. Mehrfache Aufrufe mit demselben Plugin installieren dieses nur ein einziges Mal.
```

Comment:
The phrase "hindert Dich daran" (automatically prevents you) is a little misleading since Vue probably won't raise an error, but that's what the original docs say. 🤷‍♂️